orttraining/orttraining/python/training/artifacts.py

```diff
@@ -235,8 +227,19 @@ def _export_to_ort_format(model_path, output_dir, ort_format, custom_op_library_
 
     optim_model = None
     optim_blocks = {OptimType.AdamW: onnxblock.optim.AdamW, OptimType.SGD: onnxblock.optim.SGD}
+    optim_block = None
+    if isinstance(optimizer, OptimType):
+        logging.info("Optimizer enum provided: %s", optimizer.name)
+        optim_block = optim_blocks[optimizer]()
+    elif isinstance(optimizer, onnxblock.Block):
```

Comment:
I am curious what is the motivation we allow two different types for the passed-in optimizer, specially for the onnxblock.Block ?

Reply:
The tool uses the default constructor for AdamW and SGD optimizer blocks. If users want to customize the weight_decay, or alpha or betas, they can construct the block and pass it into the tool.